web/src/app/candidate/candidate.component.html

```diff
@@ -0,0 +1,99 @@
+<div class="container">
+    <div class="panel panel-default">
+        <div class="panelheader panel-heading">Add Candidate</div>
+        <div class="panel-body">
+            <form (ngSubmit)="onSubmit(candidateForm)" #candidateForm="ngForm">
+                <div class="form-group">
+                    <div>
+                        <label>{{ 'CANDIDATE.OPENING' | translate }}</label>
+                        <div>
+                            <select [(ngModel)]="candidateModel.opening" [ngModelOptions]="{standalone: true}" required>
+                                <option disabled selected [ngValue]="null">--select--</option>
```

Comment:
add select option in the collection returned by service not here; every where;

Reply:
Done